### PR TITLE
Last fixes for v2.3.0

### PR DESCRIPTION
**Pull request by microhobby**
@andreriesco please take a look.

### CODE REVIEW

File: dotnetUno/__change__.Shared/App.xaml.cs [hunk 1]
```diff
@@ -113,10 +97,6 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
             }
 
 #if HAS_UNO_GTK
-            GtkHost.Window.Drawn += (o, args) => {
```
Comment (andreriesco) [inline dotnetUno/__change__.Shared/App.xaml.cs:116]: This part is needed for the automated tests, as it is what is printed on the console when the application runs successfully.

File: dotnetUnoFrameBuffer/.doc/README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# .NET 6 C# Uno Platform Frame Buffer template specific documentation
+# .NET 6 C# Uno Platform Fra
```
Comment (andreriesco) [inline dotnetUnoFrameBuffer/.doc/README.md]: I think here a part of the phrase has been removed: "# .NET 6 C# Uno Platform FrameBuffer template specific documentation"

File: dotnetUno/__change__.Shared/App.xaml.cs (hunk 1 above)
Comment (microhobby) [reply dotnetUno/__change__.Shared/App.xaml.cs:116]: good catch, I added a `Hello Torizon!` to handle the automated tests

File: templates.json [hunk 3]
```diff
@@ -104,15 +104,15 @@
...
-                    "amd64": ["Torizon AMD64"], 
-                    "arm": ["Torizon ARMv7"], 
+                    "local": ["Debug Local"],
+                    "amd64": ["Torizon AMD64"],
+                    "arm": ["Torizon ARMv7"],
                     "arm64": ["Torizon ARMv8"]
                 },
                 "debuggerConsole": null,
                 "mainFile": "__change__.Shared/App.xaml.cs",
                 "mainFileBreakpointLine": 117,
```
Comment (andreriesco) [inline templates.json]: Just a little update here to the new line 113

**Comment by andreriesco** [comment]
LGTM

**Pull request merged**